bin/install.sh

```diff
@@ -35,13 +35,23 @@ if [ "$?" = "1" ]; then
   exit 1
 fi
 
-echo && read -p "Would you like to use the development branch? You will get the latest features, but things may break. (y/N)" -n 1 -r -s DEV && echo
-if [ "$DEV" != 'y'  ]; then
-  BRANCH="production"
+echo && read -p "Would you like to use the experimental branch? It contains the last major changes, such as the new browser and migrating to the docker (y/N)" -n 1 -r -s EXP && echo
+if [ "$EXP" != 'y'  ]; then
+  echo && read -p "Would you like to use the development branch? You will get the latest features, but things may break. (y/N)" -n 1 -r -s DEV && echo
+  if [ "$DEV" != 'y'  ]; then
+    export DOCKER_TAG="production"
+    BRANCH="production"
+  else
+    export DOCKER_TAG="latest"
+    BRANCH="master"
+  fi
 else
-  BRANCH="master"
+  export DOCKER_TAG="latest"
```

Comment:
You're reusing the docker-tag for multiple branches.

Reply:
Yes, I suppose to use only "production" and "latest" tags , I see no reason to add others for now. Am I mistaken?

Reply:
I suggest we have the following mapping:

 * master - latest
 * production - production
 * experimental - experimental